docs/custom_quantum_functions.md

```diff
@@ -471,3 +292,15 @@ If registrations fail, then you can increase this value in your keyboard or keym
 ```c
 #define MAX_DEFERRED_EXECUTORS 16
 ```
+
+# Legacy Content :id=legacy-content
```

Comment:
Wasnt the intention to change this file and not `feature_advanced_keycodes.md`? I would somewhat expect them to be aligned, so the alternative is that the change to `feature_advanced_keycodes.md` would need to be reverted

```suggestion
# Advanced topics :id=advanced-topics
```

Reply:
Thanks !